CountlyUserDetails.m

```diff
@@ -389,7 +396,7 @@ - (void)save
     if (!CountlyConsentManager.sharedInstance.consentForUserDetails)
         return;
 
-    [CountlyConnectionManager.sharedInstance sendEvents];
+    [CountlyConnectionManager.sharedInstance sendEvents:false];
```

Comment:
and we should do this check [Countly.user isLocallyCached]
in save to eliminate any empty calls. 

And there is a possibility for the infitine recursive loop if sendEvents param injected as true

Reply:
[Countly.user isLocallyCached] is to check if there is any user properties related changes that need to send to server, we already have all the checks for user properties added in this function before sending a request.
like this way:
-- if (userDetails)
-- if (self.pictureLocalPath && !self.pictureURL)
-- if (self.modifications.count)